apps/desktop/src/lib/components/Welcome.svelte

```diff
@@ -18,19 +21,48 @@
 			newProjectLoading = false;
 		}
 	}
+
+	async function onCloneProject() {
+		cloneProjectLoading = true;
+		try {
+			goto('/onboarding/clone');
+		} finally {
+			cloneProjectLoading = false;
+		}
+	}
```

Comment:
Can there be a delay between calling `goto` and the navigation actually happening? Otherwise it feels like we don't need the `cloneProjectLoading` state here.

Reply:
Hmm yeah no probably not. This whole fn can most likely be simplified to just the goto 👍